Expensify/react-fast-pdf
# Update engines to be less strict

### Details


Updates the engines field to be a bit less strict and match `react-native-onyx`: https://github.com/Expensify/react-native-onyx/blob/main/package.json#L112-L114

### Related Issues

https://github.com/Expensify/Mobile-Expensify/pull/12655

### Manual Tests



Once we install the latest version in App we will verify `npm install` works with latest node
